apps/system/style/notifications/notifications.css

```diff
@@ -146,17 +147,19 @@
 
 #notification-bar button {
   flex: initial;
-  width: auto;
+  width: 9.5rem;
   height: 4.5rem;
-  padding: 0;
+  padding: 0 0 0 1.5rem;
   border: 0;
-  background: none;
-  color: #008eab;
```

Comment:
we should keep the background:none and the color intact for the classic rule.
we want to change the background and the color only in a 
```
#notification-bar button:active {
}
```
block